data/mods/Magiclysm/Spells/stormshaper.json

```
@@ -430,5 +431,461 @@
+    "max_aoe": 16,
+    "aoe_increment": 0.5
+  },
+  {
+    "id": "air_bubble",
+    "type": "SPELL",
+    "name": "Air Bubble",
+    "description": "Cover your head with air bubble, that allow you to breath under water or prevent breathing of toxic gases.",
+    "valid_targets": [ "self" ],
+    "flags": [ "CONCENTRATE", "LOUD", "VERBAL", "NO_LEGS" ],
+    "min_damage": 1,
+    "max_damage": 1,
+    "effect": "spawn_item",
+    "effect_str": "flask_ethereal_rebreather",
+    "shape": "blast",
+    "energy_source": "MANA",
+    "spell_class": "STORMSHAPER",
+    "difficulty": 3,
+    "max_level": 10,
+    "base_casting_time": 1000,
+    "base_energy_cost": 200,
+    "min_duration": 18000,
+    "max_duration": 180000,
+    "duration_increment": 18000
+  },
+  {
+    "id": "storm_superconductor",
+    "type": "SPELL",
+    "name": { "str": "Superconductor" },
+    "description": "Create a devastating electric charge on a small distance, roasting everything to their death.",
+    "valid_targets": [ "ally", "hostile", "ground" ],
+    "effect": "attack",
+    "effect_str": "zapped",
+    "shape": "blast",
+    "base_casting_time": 250,
+    "final_casting_time": 50,
+    "casting_time_increment": -8,
+    "base_energy_cost": 400,
+    "final_energy_cost": 150,
+    "energy_increment": -10,
+    "energy_source": "MANA",
+    "spell_class": "STORMSHAPER",
+    "difficulty": 6,
+    "max_level": 25,
+    "min_damage": 30,
+    "max_damage": 170,
+    "damage_increment": 6,
+    "damage_type": "electric",
+    "range_increment": 0.3,
+    "flags": [ "VERBAL", "LOUD", "NO_HANDS", "NO_LEGS", "RANDOM_DURATION" ],
+    "min_duration": 100,
+    "max_duration": 500,
+    "min_aoe": 1,
+    "max_aoe": 4,
+    "aoe_increment": 0.1,
+    "field_id": "fd_electricity",
+    "field_chance": 1,
+    "min_field_intensity": 1,
+    "max_field_intensity": 10,
+    "field_intensity_increment": 0.4,
+    "field_intensity_variance": 3
+  },
+  {
+    "type": "SPELL",
+    "id": "shocking_dash",
+    "name": "Shocking Dash",
+    "description": "Fill your legs with energy, and dash into picked place.",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "NO_HANDS", "LOUD" ],
+    "effect": "dash",
+    "shape": "cone",
+    "min_damage": 0,
+    "max_damage": 0,
+    "min_range": 2,
+    "max_range": 10,
+    "range_increment": 0.35,
+    "field_id": "fd_electricity",
+    "min_field_intensity": 2,
+    "max_field_intensity": 2,
+    "spell_class": "STORMSHAPER",
+    "base_casting_time": 200,
+    "final_casting_time": 50,
+    "casting_time_increment": -6,
+    "base_energy_cost": 350,
+    "final_energy_cost": 200,
+    "energy_increment": -6,
+    "max_level": 25,
+    "energy_source": "MANA",
+    "learn_spells": { "shocking_teleport": 15 }
+  },
+  {
+    "type": "SPELL",
+    "id": "shocking_teleport",
+    "name": "Shocking Teleport",
+    "description": "Fill your legs with energy, and dash into picked place, ignoring all obstacles in between.",
```

```suggestion
    "description": "Briefly become lightning to teleport to a chosen spot, ignoring all obstacles in-between.",
```
If this is a teleport rather than a dash, more of the body should be affected.